tests/bootstrap.php

```diff
@@ -7,6 +7,17 @@
 require_once __DIR__ . '/../wp-includes/sqlite/class-wp-sqlite-pdo-user-defined-functions.php';
 require_once __DIR__ . '/../wp-includes/sqlite/class-wp-sqlite-translator.php';
 
+/**
+ * Polyfills for WordPress functions
+ */
+
+function do_action( $tag, ...$args ) {
```

Comment:
Unit tests crashed for me without these, I'm not sure how to run them.

Reply:
I think it's safe to keep them in this case...